source/MRIOExtras/MRPdf.h

```diff
@@ -79,6 +80,9 @@ class Pdf
     MRIOEXTRAS_API void addImageFromFile( const std::filesystem::path& imagePath, const std::string& caption = {},
         const std::vector<std::pair<double, std::string>>& valuesMarks = {} );
 
+    // simple method to add images from file
+    MRIOEXTRAS_API void addImageFromFileSimple( const std::filesystem::path& imagePath, const Vector2f& pos, const Vector2f& size );
```

Comment:
the method is named "simple", but it takes more parameters (pos, size), than not-simple one. Can we have just one `addImageFromFile` with parameters supplied in a struct?